extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -225,9 +225,14 @@ export class AmpAnalytics extends AMP.BaseElement {
         this.instrumentation_.createAnalyticsGroup(this.element);
 
     if (this.config_['transport'] && this.config_['transport']['iframe']) {
+      const ampAdResourceId = user().assertString(
+          getAmpAdResourceId(this.element, this.win.top),
+          `${TAG}: No friendly parent amp-ad element was found for ` +
```

Comment:
can we be more specific in the error message. Please mention the use case here (transport: iframe). no friendly parent amp-ad element is valid with all other transport type.

Reply:
Done.